packages/storybook-ui-components/stories/2-Icon.stories.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import { withKnobs, select } from "@storybook/addon-knobs";
+
+import { Icon } from "@cockroachlabs/ui-components";
+
+export default {
+  title: "Icon",
+  component: Icon,
+  decorators: [withKnobs],
+};
+
+const IconDisplaySection = ({ children }) => (
+  <section
+    style={{
```

Comment:
Would be great if we were able to share these styles across stories so we could have consistent layouts. Might be worth a ticket if it doesn't already exist.

Reply:
Anne and Rachel are working on _standardizing_ what our stories should look like. I agree with you about these ad hoc inline styles. I'm hoping their work will resolve the inconsistencies in the stories as well as the README conversations we've been having all over the place.

All that to say, I'm punting for now.